### ISSUE TITLE
Create/Edit Org Page:  Admin - Error handling

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#2438]
When submitting the form without all fields, an error message is briefly shown at the bottom before the org page is loaded.

We will need to align the form validation and error messages with what is used for other forms in the site.

![image](https://cloud.githubusercontent.com/assets/1654485/6665997/41b1e47c-cbe0-11e4-8d8f-4f7c0f594880.png)

**Comment by danmihaila** [comment]
also there is a message saying that "you are going to leave this page" are you sure?
moving this to sprint candidates

**Comment by mbellotti** [comment]
This doesn't have anything to do with the admin page. It happens because the mailing list form field is empty.

**Comment by cjhendrix** [comment]
We assigned it to you anyway.  :)

**Comment by danmihaila** [comment]
About those 2 errors:
A. mandatory field red alert: 
  that alert is raised when "update organization" button is clicked, so it has to do with form. My guess is  we have 2 forms with the same or a form tag is not closed.

B. Javascript Alert
I tried to reproduce it:
1. I edited wfp org
2. add logos
3. add vizualization type (wfp and not 3w as selected by default)

I got this alert: check screenshot
![screen shot 2015-04-02 at 14 04 01](https://cloud.githubusercontent.com/assets/3865844/6963458/62584232-d948-11e4-97d4-9af64a8473a4.png)
